src/deviation.jl

```diff
@@ -12,7 +12,7 @@ function counteq(a::AbstractArray, b::AbstractArray)
     n = length(a)
     length(b) == n || throw(DimensionMismatch("Inconsistent lengths."))
     c = 0
-    for i = 1:n
+    for i in eachindex(a, b)
```

Comment:
Thanks for doing this (it's been long due!), but can you move this to another PR?

Reply:
Yup, I have done it...